common/views/components/EventPromo/README.md

```diff
@@ -0,0 +1,7 @@
+Event Promo
+
+image 16:9
```

Comment:
Could we use consistent language across the READMEs for this?

Reply:
yep, but this was really just a placeholder - think we need to discuss as a team what is useful to go in here